notebooks/segmentation/segmentation_inspection.ipynb

```diff
@@ -327,15 +427,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%time\n",
-    "Video(**display_generator.display_video(padded_roi_thumbnail, width=512, height=512))"
+    "Video(**display_generator.display_video(denoised_video_with_roi))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Display without the ROI's border"
+    "It is also possible to generate a thumbnail video by specifying an origin and a field of view shape (this thumbnail video will not have any ROIs overplotted)."
```

Comment:
I love this part.  If the tools provided above don't meet their needs, this part is flexible enough for them to do whatever they want. 

The one thing that may be worth adding, is that the `,timesteps=timesteps` kwarg from above can also be added here so that the user can limit to a particular time segment of the video